src/components/qas/questions/qaCalculators/kineticEnergyCalculator/kineticEnergyCalculator.module.scss

```diff
@@ -0,0 +1,77 @@
+:global {
```

Comment:
again, I think this is a misuse of the global flag.  If it' use is unclear I'm happy to talk about it together or feel free to [spend some time](https://github.com/css-modules/css-modules/issues/264#issuecomment-609067219) reading up [about it](https://blog.yipl.com.np/css-modules-with-react-the-complete-guide-a98737f79c7c) We should try really hard to use this sparingly, ONLY for classes outside our control, and, if possible, scope it under a class we DO have control over (although I know that's not always gonna be possible).  I recognize there are many of those classes outside our control in this component, but you also have a lot of "our" classes inside the global flag too.  Those are gonna pollute our styles scope and, if used across like components, with like class names, will potentially cause unintended consequences.  I actually ran into just that problem recently and so cleaned up quite a bit of this kind of thing in the other calculator components.  Please look to those components for some indications of how to address the issue here in this file.

Additionally, I'm seeing a lot of unused classes in here.  Please delete anything that's not being used.  It's really important to eliminate unused code like this (and similar in javascript files) before we commit work to master because it visually clutters up our files, increases bloat, and can really hard to spot later on when this code is not fresh in our minds.